indico/modules/events/papers/client/js/components/PaperDecisionForm.jsx

```diff
@@ -58,6 +57,15 @@ export default function PaperDecisionForm() {
     return null;
   }
 
+  if (state.name === PaperState.submitted && !actionOptions[PaperState.to_be_corrected]) {
+    actionOptions[PaperState.to_be_corrected] = {
+      value: 'to_be_corrected',
+      text: Translate.string('To be corrected'),
+    };
+  } else if (state.name === PaperState.to_be_corrected) {
+    delete actionOptions[PaperState.to_be_corrected];
+  }
```

Comment:
Modifying an object that's not defined inside the component feels pretty dirty.. maybe better to just define it inside?